x/monitoringc/client/cli/tx_create_client.go

```diff
@@ -11,12 +11,22 @@ import (
 	"github.com/tendermint/spn/x/monitoringc/types"
 )
 
+const (
+	flagUnbondingPeriod = "unbonding-period"
+	flagRevisionHeight  = "revision-height"
+)
+
 func CmdCreateClient() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "create-client [launch-id] [consensus-state-file] [validator-set-file]",
 		Short: "Create a verified client ID to connect to the chain with the specified launch ID",
 		Args:  cobra.ExactArgs(3),
 		RunE: func(cmd *cobra.Command, args []string) (err error) {
+			var (
+				unbondingTime, _  = cmd.Flags().GetInt64(flagUnbondingPeriod)
+				revisionHeight, _ = cmd.Flags().GetUint64(flagRevisionHeight)
+			)
```

Comment:
```suggestion
			unbondingTime, _  := cmd.Flags().GetInt64(flagUnbondingPeriod)
			revisionHeight, _ := cmd.Flags().GetUint64(flagRevisionHeight)
```

Reply:
Also, shouldn't we check explicitly for errors instead of ignoring them?

Reply:
They are programmer errors, we don't check them for simplicity

Reply:
sounds good! I was just wondering and thought to ask.